sphinx/source/docs/user_guide/annotations.rst

```diff
@@ -310,7 +310,7 @@ single point would be one common use for the Whisker annotation.
 
 .. |Plot| replace:: :class:`~bokeh.models.plots.Plot`
 
-.. |Figure| replace:: :class:`~bokeh.plotting.Figure`
+.. |Figure| replace:: :class:`~bokeh.plotting.figure.Figure`
```

Comment:
I'm surprised this is necessary. As an aside the submodules under `bokeh.plotting` should probably be made private.

Reply:
See below...